### ISSUE TITLE
feature idea: use externally of project

### ISSUE THREAD

**Issue by eccegordo** [seawatts/ember-cli-commands#49]
We should consider the use case of using this without being installed inside a local ember project or addon.

imagine it being installed globally and then upgrade the deps of any project on your computer.

I suggest we create a separate command for this.

```
ember upgrade-project --path=path/to/my/app
```

It could probably share some of the other commands and flags. 

The use case is I may have numerous small projects on my computer and I just want to install this once to run it. Plus I may have reasons for not including as a dependency on all my projects, particularly if I want a really light weight project, or CI reasons for keeping the dep tree as minimal as possible. Plus it is one more thing to keep updated if I have it installed in many different projects.

**Comment by seawatts** [comment]
Yeah I like the idea. I think if we can get this pulled into ember-cli proper then we will get this feature for free

**Comment by eccegordo** [comment]
I am definitely curious to understand how that will work. I suppose it will just be part of the global ember cli installed and we point it to any project we want.
